estate/models/estate_property.py

```diff
@@ -0,0 +1,67 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+from datetime import timedelta
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Estate Property'
+
+    name = fields.Char('Title', required=True, default='Unknown', translate='True')
+    active = fields.Boolean('Active', default=True)
+    description = fields.Text('Description')
+    postcode = fields.Char('Postcode')
+    date_availability = fields.Date('Available From', copy=False, default=fields.Datetime.now() + timedelta(days=90))
```

Comment:
```suggestion
    date_availability = fields.Date('Available From', copy=False, default=fields.Date.add(fields.Date.today(), months=3)
```
What you are doing is correct as well but we have a helper in odoo to do the same so it is better to use it